packages/k2-alpine/src/theme/tokens/text.ts

```diff
@@ -0,0 +1,154 @@
+// K2 Foundation - Typography
+// https://www.figma.com/file/hDSl4OUgXorDAtqPZtCUhB/K2-Foundation?type=design&node-id=3390-22100
+
+export const text = {
+  heading1: {
+    fontFamily: 'Inter-Regular',
+    fontSize: 60,
+    lineHeight: 72
+  },
+  heading2: {
+    fontFamily: 'Inter-Regular',
```

Comment:
are we just reusing the same typography? I thought there would be new fonts?

Reply:
I grabbed the old file content just to have some placeholder for this token. I haven't implemented anything with these yet.

Reply:
looks like it's now ready here https://www.figma.com/design/opZ4C1UGzcoGRjxE4ZIE3J/K2-Alpine?node-id=14-4148&node-type=CANVAS&t=M0oUVuMWoJ86MnPG-0. can you add all the mobile ones here? the font is Aeonik.

Reply:
the work will be done in https://ava-labs.atlassian.net/browse/CP-9162